qiskit/quantum_info/operators/operator.py

```diff
@@ -615,6 +615,62 @@ def _multiply(self, other):
         ret._data = other * self._data
         return ret
 
+    def _equal_with_ancillas(
+        self,
+        other: Operator,
+        ancilla_qubits: list,
+        ignore_phase: bool = False,
+        rtol: float | None = None,
+        atol: float | None = None,
+    ) -> bool:
+        r"""Test if two Operators are equal on the subspace where ancilla qubits
+        are :math:`|0\rangle`.
+
+        Args:
+            other (Operator): an operator object.
+            ancilla_qubits: a list of clean ancilla qubits.
+            rtol (float): relative tolerance value for comparison.
+            atol (float): absolute tolerance value for comparison.
+            ignore_phase (bool): ignore complex-phase difference between matrices.
+
+        Returns:
+            bool: True if operators are equal up to clean ancilla qubits.
```

Comment:
```suggestion
            bool: True iff operators are equal up to clean ancilla qubits.
```

Reply:
Done in [4de623a](https://github.com/Qiskit/qiskit/pull/12968/commits/4de623a146e16e451e1afadd6f4ef12e27d78402).